admin/email-editor-metabox.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Handles creating the email editor metaboxes
+ **/
+if ( ! class_exists( 'UCF_Email_Editor_Metabox' ) ) {
+	class UCF_Email_Editor_Metabox {
+
+
+		/**
+		 * Adds the Custom Email Editor metabox
+		 * @author RJ Bruneel
+		 * @since 1.0.0
+		 **/
+		public static function add_meta_box() {
+			add_meta_box(
+				'ucf-email-editor',
+				__( 'Email Signature' ),
+				array( 'UCF_Email_Editor_Metabox', 'metabox_markup' ),
+				'ucf-email'
+			);
+		}
+
+
+		/**
+		 * The markup callback for the Email Editor metabox
+		 * @author RJ Bruneel
+		 * @since 1.0.0
+		 * @param $post WP_Post | The current post object
+		 * @return string | The function output is echoed
+		 **/
+		public static function metabox_markup( $post ) {
+			wp_nonce_field(  'ucf_email_editor_nonce_save', 'ucf_email_editor_nonce' );
+			$value = get_post_meta( $post->ID, 'ucf_email_editor_signature', true );
+			$value = ( empty( $value ) ) ? 'president' : $value;
+		?>
+			<table class="form-table">
+				<tbody>
+					<tr>
+						<th><strong>Select an Email Signature</strong></th>
+						<td>
+						<input type="radio" name="ucf_email_editor_signature" value="president" <?php checked( $value, 'president' ); ?> >President<br>
```

Comment:
Wrapping each of these `<input>`'s in a `<label>` would make them easier to select